.github/workflows/dev-aws-ecr.yml

```diff
@@ -57,7 +57,7 @@ jobs:
         CERT: ${{ secrets.SAML_CERT }}
 
       run: |
-        sudo echo $CERT > /usr/local/share/ca-certificates/federation.dev.cce.af.mil.crt
+        echo $CERT > saml.pem
         sudo sh scripts/add-dod-cas.sh
```

Comment:
Does this script run twice, once here and once in the Dockerfile?

Reply:
Yes. In the Dockerfile `saml.pem` is necessary for creating the GCDS chain. `add-dod-cas.sh` is called in the Dockerfile to also create the chain and overall, just exist for other DoD systems in the future. 

`add-dod-cas.sh` also runs on the Github Runner, because we need it so we can `docker [login/push]` to artifactory which uses a self-signed cert that's issued by DoD CA's.